### PR TITLE
Add the OverrideParser class and tests.

### PR DESCRIPTION
**Pull request by corthon**

### CODE REVIEW

**Comment by corthon** [comment]
@matthewfcarlson, this is the PR I mentioned last week, with the unit tests.

File: MuPythonLibrary/Uefi/EdkII/Parsers/OverrideParser.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
+## @file OverrideParser.py
+# Contains classes to help with the parsing of INF files that
+# make contain OVERRIDE information.
```
Comment (spbrogan) [inline MuPythonLibrary/Uefi/EdkII/Parsers/OverrideParser.py]: may??

File: MuPythonLibrary/Uefi/EdkII/Parsers/OverrideParser.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        self.overrides = []
+        for override_line in self.override_lines:
+            try:
+                self.overrides.append(self._parse_override_line(override_line['line']))
+            except OpParseError as pe:
+                raise ValueError("Parse error '%s' occurred while processing line %d of '%s'." %
+                                 (pe, override_line['lineno'], override_line['line']))
+
+    @staticmethod
+    def _get_override_lines(parse_contents):
```
Comment (spbrogan) [inline MuPythonLibrary/Uefi/EdkII/Parsers/OverrideParser.py]: should not be private (leading with "_")

**Review by spbrogan — APPROVED** [review]

**Review by matthewfcarlson — APPROVED** [review]
Looks great

**Pull request merged**